.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,46 @@
+// For format details, see https://aka.ms/vscode-remote/devcontainer.json or this file's README at:
```

Comment:
Yes, this is an auto generated file that supports opening/debugging in a container in visual studio code. Sharing this file in the repo lets other VS Code users connect to eh dev container with a couple of clicks.

Reply:
I don't care to have editor specific code in a repo personally, but since so much of the team uses VS Code I guess there is an argument for it. Is it possible to separate this code into a separate repo to have shared vscode config or is this config really specific to this project?

Reply:
The best practice seems to be to include it with the project, they are specific to the project. If you don't use VS Code it should have no impact though I understand not wanting something this specific in the repo. I think this works toward the goal of providing a "template" for devs to get up and running quickly, and including this enables a nice debugger in VS Code.

Reply:
I understand the goal, and I am supportive. It just seems that it should be possible to standardize on the setup, and place it outside the repo, so that the setup can be shared among our different projects. Something to think about.

Reply:
Will do. If nothing else comes up, maybe I can demo this on Friday and we can try to figure out next steps.